### PR TITLE
[zh-cn] Small fix about cri page.

### PR DESCRIPTION
**Pull request by qlijin**
Small fix about CRI page:
 content/zh-cn/docs/concepts/architecture/cri.md
 content/zh-cn/docs/reference/glossary/container-runtime-interface.md

### CODE REVIEW

**Comment by windsonsea** [comment]
Both files show in sync when you run `./scripts/lsync.sh content/zh-cn/docs`, you may need to confirm all changes have been synced in this PR.

**Comment by qlijin** [comment]
OK

**Comment by qlijin** [comment]
Is the lsync.sh has a bug? 

Because the result of directory and file is diffrence:
微信截图_20221228112027

The code below shoude be changed like this: 
`if [ "$?" -ne 0 ]; then `

not the code below:
https://github.com/kubernetes/website/blob/34732f266836de6d5c52702a0d8b39bbc50dc829/scripts/lsync.sh#L57

File: content/zh-cn/docs/concepts/architecture/cri.md [hunk 1]
```diff
@@ -58,8 +58,9 @@ If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
 and doesn't register as a node.
 -->
 对 Kubernetes v{{< skew currentVersion >}}，kubelet 偏向于使用 CRI `v1` 版本。
-如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商任何旧的其他支持版本。
-如果 kubelet 无法协商支持的 CRI 版本，则 kubelet 放弃并且不会注册为节点。
+如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商任何更旧的可支持版本。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/cri.md:61]:
```suggestion
如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商较老的、仍被支持的所有版本。
```

File: content/zh-cn/docs/concepts/architecture/cri.md [hunk 2]
```diff
@@ -58,8 +58,9 @@ If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
...
+v{{< skew currentVersion >}}版本中kubelet也可协商 CRI `v1alpha2` 版本，但此版本被视为已弃用。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/cri.md:62]:
```suggestion
v{{< skew currentVersion >}} 版本的 kubelet 也可协商 CRI `v1alpha2` 版本，但该版本被视为已弃用。 
```

File: content/zh-cn/docs/concepts/architecture/cri.md [hunk 3]
```diff
@@ -58,8 +58,9 @@ If the kubelet cannot negotiate a supported CRI version, the kubelet gives up
 and doesn't register as a node.
 -->
 对 Kubernetes v{{< skew currentVersion >}}，kubelet 偏向于使用 CRI `v1` 版本。
-如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商任何旧的其他支持版本。
-如果 kubelet 无法协商支持的 CRI 版本，则 kubelet 放弃并且不会注册为节点。
+如果容器运行时不支持 CRI 的 `v1` 版本，那么 kubelet 会尝试协商任何更旧的可支持版本。
+v{{< skew currentVersion >}}版本中kubelet也可协商 CRI `v1alpha2` 版本，但此版本被视为已弃用。 
+如果 kubelet 无法协商可支持的 CRI 版本，则 kubelet 放弃并且不会注册为节点。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/cri.md:63]:
```suggestion
如果 kubelet 无法协商出可支持的 CRI 版本，则 kubelet 放弃并且不会注册为节点。
```

**Review by my-git9 — COMMENTED** [review]
/lgtm

**Pull request merged**